app/src/main/java/org/wikipedia/descriptions/DescriptionEditReadArticleBarView.kt

```diff
@@ -34,9 +39,31 @@ class DescriptionEditReadArticleBarView @JvmOverloads constructor(
             viewArticleImage!!.visibility = GONE
         } else {
             viewArticleImage!!.visibility = VISIBLE
-            viewArticleImage!!.loadImage(Uri.parse(pageSummary.thumbnailUrl))
+            ViewUtil.loadImageUrlInto(viewArticleImage!!, pageSummary.thumbnailUrl)
+        }
+        viewImageThumbnail.visibility = View.GONE
+        viewReadButton.visibility = View.VISIBLE
+        show()
+    }
+
+    fun setImageDetails(thumbUrl: String, fileName: String) {
+        setConditionalLayoutDirection(this, WikipediaApp.getInstance().language().appLanguageCode)
```

Comment:
Just note that the layout direction should rely on the language code from ImageDetails data.

Reply:
I will just make a note for now..and change it with other changes